README.md

```diff
@@ -14,33 +14,41 @@
 8. [User Stories](./docs/userStories.md)
 
 ## <a name="Technologies"></a>Technologies
-express
-mongoose
+- Node
+- Express
+- Mongo
+- React
+- Webpack
+- Mongoose
+- jQuery
+- P5.js
 
 ## <a name="File-Structure"></a>File-Structure
 dreamers/
   .git/
   docs/
   dreamersApp/
-    build/
+    node_modules/
+    public/
       static/
         css/
         js/
         media/
-    public/
     src/
+  dreamy-backend/
+    node_modules/
   .gitignore
   README.md
 
```

Comment:
Are these extra whitespaces a problem?  Should they be deleted?

Reply:
Removed these spaces